package.json

```diff
@@ -38,10 +38,10 @@
     "lerna": "^3.16.4",
     "lint-staged": "^9.5.0",
     "prettier": "^1.19.1",
-    "rimraf": "^2.6.3",
+    "rimraf": "~3.0.0",
```

Comment:
should this be "~3.0.2" to match rimraf version in packages/application/package.json ?

Reply:
actually its the opposite, application is the one that's different than the rest, I'll catch this in my upcoming commit

Reply:
yes, I updated all to use `"rimraf": "~3.0.0"`